source/_integrations/adaptive_lighting.markdown

```diff
@@ -0,0 +1,240 @@
+  required: inclusive
+  default: false
+  type: boolean
+{% endconfiguration %}
+
+Full example:
+
+```yaml
+# Example configuration.yaml entry
+adaptive_lighting:
+- name: "default"
+  lights: []
+  prefer_rgb_color: false
+  transition: 45
+  initial_transition: 1
+  interval: 90
+  min_brightness: 1
+  max_brightness: 100
+  min_color_temp: 2000
+  max_color_temp: 5500
+  sleep_brightness: 1
+  sleep_color_temp: 1000
+  sunrise_time: "08:00:00"  # override the sunrise time
+  sunrise_offset:
+  sunset_time:
+  sunset_offset: 1800  # in seconds or '00:15:00'
+  take_over_control: true
+  detect_non_ha_changes: false
+  only_once: false
+
+```
+
+### Services
+
+`adaptive_lighting.apply` applies Adaptive Lighting settings to lights on demand.
+
+| Service data attribute    | Optional | Description                                                             |
+|---------------------------|----------|-------------------------------------------------------------------------|
+| `entity_id`               |       no | The `entity_id` of the switch with the settings to apply.               |
+| `lights`                  |       no | A light (or list of lights) to apply the settings to.                   |
+| `transition`              |      yes | The number of seconds for the transition.                               |
+| `adapt_brightness` | yes | Whether to change the brightness of the light or not.                                        |
+| `adapt_color`      | yes | Whether to adapt the color on supporting lights.                                             |
+| `prefer_rgb_color` | yes | Whether to prefer RGB color adjustment over of native light color temperature when possible. |
+| `turn_on_lights`   | yes | Whether to turn on lights that are currently off.                                            |
+
+`adaptive_lighting.set_manual_control` can mark (or unmark) whether a light is "manually controlled", meaning that when a light has `manual_control`, the light is not adapted.
+
+| Service data attribute | Optional | Description                                                                                                                          |
+|------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------|
+| `entity_id`            |       no | The `entity_id` of the switch in which to (un)mark the light as being "manually controlled".                                         |
+| `lights`               |       no | A light (or list of lights) to apply the settings to.                                                                                |
+| `manual_control`       |       no | Whether to mark (true) or unmark (false) the light as "manually controlled", when not specified it selects all lights in the switch. |
```

Comment:
```suggestion
| `manual_control`       |      yes | Whether to mark (true) or unmark (false) the light as "manually controlled", when not specified it selects all lights in the switch. Default: true |
```

Reply:
Couldn't we add another column for defaults?

Reply:
Also on your next one, drag the plus button to suggest changes to multiple lines at once.